README.md

```diff
@@ -33,49 +39,72 @@ This repository is organized as follows:
     ├── svc-[service].yml   # defines an individual service api
     ├── quartz-oem.yml      # defines the oem quartz api
     └── unity.yml           # defines the unity api
+
 ```
 
-When adding a service API definition, add the service specific components to a subdirectory inside `src/svc` and reference them from a file in `src` with a prefix `svc-`. This allows product API maintainers to copy the service-specific ("internal") paths and components into the respective API definition (cloud, cloud-priv, or oss) without modifying references. For more information on how to add a new service api defnition, [look here](./src/svc/README.md). For information on what to do when the platform APIs drift, [look here](./src/README.md).
+## How to contribute
+
+After you make changes, run `make generate-all` (`docker` currently required) to generate new API contracts.
+Commit the result, including the updated `./contracts` files, and create a pull request.
+
+Use the following guidelines when adding or updating specifications.
+
+### Paths and operations
+
+If you're adding or updating paths and operations (`get`, `post`, etc.), follow our [path template](./docs/templates/pathTemplate.yml) for `summary`, `description`, and `example` elements.
+
+### Service definitions
+
+To add a service API definition, add the service specific components to a subdirectory inside `src/svc` and reference them from a file in `src` with a prefix `svc-`. This allows product API maintainers to copy the service-specific ("internal") paths and components into the respective API definition (cloud, cloud-priv, or oss) without modifying references. For more information on how to add a new service api defnition, see [src/svc/README](./src/svc/README.md).
```

Comment:
do we want to recommend naming conventions for the subdirectory?

Reply:
Do we have an example service to model? (maybe that's referred to in the svc README)